test/mCODECancerDiseaseStatus.test.js

```diff
@@ -12,36 +10,33 @@ beforeAll(() => {
 
   const patientBundle = loadJSONFixture(path.join(__dirname, './fixtures/patients/Bundle-mCODECQLExample01.json'));
 
-  testSetup = setup('mCODECancerDiseaseStatusTest', elm, patientBundle, valueSetMap);
+  executionResults = execute(elm, [patientBundle], valueSetMap, 'mCODECancerDiseaseStatusTest');
```

Comment:
Very minor thing, but the execute function can take either a single patient or an array of patients, so you don't technically need the brackets here but it does still work. Figured I'd point it out because you pass a single patient without brackets in other tests, might be good to keep it consistent